backend/src/packages/chaiNNer_standard/image_filter/noise/fast_nlmeans.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import annotations
+
+import numpy as np
+import cv2
+
+from nodes.properties.inputs import ImageInput, SliderInput, NumberInput
+from nodes.properties.outputs import ImageOutput
+from nodes.groups import Condition, if_group
+from nodes.impl.image_utils import to_uint8
+from nodes.utils.utils import get_h_w_c
+
+from .. import noise_group
+
+
+@noise_group.register(
+    schema_id="chainner:image:fast_nlmeans",
+    name="Fast NL means",
+    description="Use the fast Non-Local Means algorithm to denoise an image.",
+    icon="CgEditNoise",
+    inputs=[
+        ImageInput("Image", channels=[1, 3, 4]),
+        SliderInput(
+            "Luminance strength",
+            minimum=0.,
+            maximum=50.,
```

Comment:
We don't care whether those values are floats or ints. They will be sent to the frontend via JSON, so this information is lost during the JSON conversion.

However, I think that extra dot is annoying, so could we please get rid of it?

```suggestion
            minimum=0,
            maximum=50,
```